app/serializers/serialize_exercise_representations.rb

```diff
@@ -32,6 +32,9 @@ def call
         },
         num_submissions: representation.num_submissions,
         appears_frequently: representation.appears_frequently?,
+        author: User.where(id: representation.feedback_author_id).pick(:handle),
+        editor: User.where(id: representation.feedback_editor_id).pick(:handle),
```

Comment:
I would change this to:

```suggestion
        author: representation.feedback_author.handle,
        editor: representation.feedback_editor.handle,
```

Your version will do two queries for each representation, whereas the suggested code could use `.includes` (you should get a warning) or `joins`